util/commands.sh

```diff
@@ -6,9 +6,9 @@ function wptd_chown() {
   docker exec -u 0:0 "${DOCKER_INSTANCE}" chown -R $(id -u $USER):$(id -g $USER) $1
 }
 function wptd_useradd() {
-  docker exec -u 0:0 "${DOCKER_INSTANCE}" groupadd -g "$(id -g $USER)" user
-  docker exec -u 0:0 "${DOCKER_INSTANCE}" useradd -u "$(id -u $USER)" -g "$(id -g $USER)" user
-  docker exec -u 0:0 "${DOCKER_INSTANCE}" usermod -a -G user root
```

Comment:
I see you've dropped root here - how confident are we that this doesn't break some not-used-by-travis rules?

Reply:
Quite confident. I'll revert this if anything breaks :)

And I'll make it clear in the commit message.